src/hotspot/share/oops/instanceKlass.inline.hpp

```diff
@@ -185,16 +185,4 @@ ALWAYSINLINE void InstanceKlass::oop_oop_iterate_bounded(oop obj, OopClosureType
   oop_oop_iterate_oop_maps_bounded<T>(obj, closure, mr);
 }
 
-inline instanceOop InstanceKlass::allocate_instance(oop java_class, TRAPS) {
```

Comment:
In moving this, can you eliminate any #includes at the top?  And migrate them to .cpp files that might need them.

Reply:
All the #include headers are removed except the `utilities/devirtualizer.inline.hpp`. The .cpp files that need it should include it before `oops/instanceKlass.inline.hpp` which violates the _sorted header filename_ in the coding style rules.

Reply:
Good.  I think devirtualizer.inline.hpp is used by the GC inlined functions so should remain here.

Reply:
Sorry for being confusing before.  My comment was that you should remove #includes that aren't directly referenced by this .inline.hpp file.  I see that you've done this now, but do you need classfile/vmSymbols.hpp here?  I don't see it.

Reply:
I mis-interpreted the required changes.
It is not needed here, and moved to `jvmtiEnvBase.cpp`.